website/blog/2025-01-28-git-branching-strategies-and-dbt.md

```diff
@@ -460,12 +461,11 @@ This is an action performed outside of dbt through git operations - however, we
 <summary>How do you make releases?</summary>
 <p>
 
-For our examples, a release is just a pull request to get changes into `main` from `qa`, opened from the git platform. 
+In our examples, we noted that our definition of a release is a pull request from `qa` to `main`, and this is opened from the git platform. 
 
-**You should be aware that having the source branch as `qa` on your pull request will also incorporate any new merges to `qa` since you opened the pull request, until it’s merged.** Because of this it’s important that the person opening a release is aware of what the latest changes were and when a job last ran to indicate the success of all the release’s changes. There are two options we like to implement to make this easier\:
-
-- A CI job for pull requests to `main` - this will catch and rerun our CI job if there’s any new commits on our `qa` branch
-- An [on-merge job](https://docs.getdbt.com/docs/deploy/merge-jobs) using our `qa` environment. This will run a job any time someone merges. You may opt for this if you’d rather not wait on a CI pipeline to finish when you open a release. If this option is used, the latest job that ran should be successful and linked on the release’s PR.
+**Having the source branch as `qa` on your pull request will also incorporate any new merges to `qa` while your PR stays open, possibly resulting in other features being promoted to `main` unintentionally once you merge.** Because of this it’s important that the person opening a release stays up to date on merges and last runs to ensure the validity of changes before the release is merged. There are two options we like to implement to make this easier\:
```

Comment:
```suggestion
**Having the source branch as `qa` on your pull request will also incorporate any new merges to `qa` while your PR stays open, possibly resulting in other features being promoted to `main` unintentionally once you merge.** Because of this, it’s important that the person opening a release stays up to date on merges and last runs to ensure the validity of changes before the release is merged. There are two options we like to implement to make this easier\:
```